modules/oadp-ssec-encrypted-backups.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-ssec-encrypted-backups_{context}"]
+= Creating a OADP SSE-C encryption key for additional data security
+
+{aws-first} S3 now applies server-side encryption with Amazon S3 managed keys (SSE-S3) as the base level of encryption for every bucket in Amazon S3.
+
+Note that {oadp-first} encrypts data by using SSL/TLS, HTTPS, and uses the `velero-repo-credentials` secret when transferring the data from the cluster to storage. To avoid backup data from being exposed if you lose your AWS credentials or they are stolen, it is important to include an additional layer of encryption.
+
+The _velero-plugin-for-aws_ provides several additional encryption methods. You should review its configuration options and consider implementing additional encryption.
```

Comment:
Verify the need to italicize this. It wouldn't typically be per ISG. Also, awk, but I believe the currently italicized term would need to be an adjective with this construction, so:

* The velero-plugin-for-aws plugin provides...
* velero-plugin-for-aws provides...